integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktApplication.java

```diff
@@ -33,40 +32,35 @@ public ServletRegistrationBean servletNoArk() {
 
     public static void main(String[] args) {
         try {
-            if(!validateJCE()){
+            if (!validateJCE()) {
                 logMissingJCE();
                 return;
             }
-            
-            ConfigurableApplicationContext context = SpringApplication.run(IntegrasjonspunktApplication.class, args);
         } catch (SecurityException se) {
             logMissingJCE(se);
         }
+        try (ConfigurableApplicationContext context = SpringApplication.run(IntegrasjonspunktApplication.class, args)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00108)